### DISCUSSION TITLE
Query related to heat conduction time derivative

### DISCUSSION THREAD

**Discussion by deepakbiitk** [idaholab/moose#24746] [Q&A General] [answered]
Hi all,

I am solving a transient heat equation, looking through the warning of HeatConductionTimeDerivative (https://mooseframework.inl.gov/source/kernels/HeatConductionTimeDerivative.html). Do it mean it creates a lumped matrix?

Thanks,
Deepak

**Comment by GiudGiud** [comment, marked as answer]
The warning is because this kernel is missing off-diagonal terms for the Jacobian
